src/components/Player/Player.vue

```diff
@@ -26,7 +26,6 @@
       :class="{ 'mt-20': stream.isVideo }"
       :isVideo="stream.isVideo"
       :player="player"
-      :autoplay.sync="autoplay"
```

Comment:
This change snuck in -- `autoplay` is no longer defined on MediaControls, so this code doesn't do anything and I've removed it.